### PR TITLE
4/13 cake - Chocolatey Cake

### PR DESCRIPTION
**Pull request by glubtier**
Based on the cake from Matilda.

Item texture: Rose
Block textures: Vinelli
Code: glubtier

### CODE REVIEW

**Comment by glubtier** [comment]
Also -- if someone wants to make changes to things like distribution, name, add them to consort loot tables, etc., have at it!

File: src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java
Comment (glubtier) [reply src/main/java/com/mraof/minestuck/data/recipe/MinestuckCombinationsProvider.java:1063]: Whoops yep, wasn't intentional, just forgot. Added.

**Review by kirderf1 — APPROVED** [review]

File: CHANGELOG.md
Comment (Dweblenod) [inline CHANGELOG.md:38]: add a space between this line and ### Changed

**Review by Dweblenod — APPROVED** [review]
Once the changelog fix is done, this has my approval!

File: CHANGELOG.md
Comment (glubtier) [reply CHANGELOG.md]: Done

**Pull request merged**
